theme/n21/static/scripts/n21-blog.js

```diff
@@ -0,0 +1,19 @@
+const stripHtml = require('string-strip-html');
+
+$(document).ready(() => {
+	$.ajax({
+		url: '/ghost/startseite-nbc',
+		type: 'GET',
+		dataType: 'json',
+		contentType: 'application/json',
+		timeout: 8000,
+	})
+		.done((result) => {
+			$('.n21-blog .title').text(result.pages[0].title);
+			$('.n21-blog .content').html(stripHtml(result.pages[0].html, { onlyStripTags: ['script', 'style'] }));
+		})
+		.fail(() => {
+			/* eslint-disable-next-line */
+			console.error('Could not load frontpage content from blog');
```

Comment:
Here instead show a placeholder on the page and possibly add a manual reload functionality